plugins/statistics.koplugin/main.lua

```diff
@@ -1498,7 +1520,19 @@ function ReaderStatistics:getCurrentStat()
                 }
                 UIManager:show(self.kv)
             end,
-        }
+        },
+        { _("Average time per day"), datetime.secondsToClockDuration(user_duration_format, book_read_time/tonumber(total_days), false, true), separator = true },
+
+        -- Book progression
+        { _("Book start date"), T(N_("(1 day ago) %2", "(%1 days ago) %2", first_open_days_ago), first_open_days_ago, datetime.secondsToDate(tonumber(first_open), true)) },
+        { _("Estimated finish date"), estimates_valid and T(N_("(in 1 day) %2", "(in %1 days) %2", estimate_days_to_read), estimate_days_to_read, estimate_end_of_read_date) or _("N/A") },
+        -- estimation, from current page to end of book
+        { _("Estimated time left"), estimates_valid and datetime.secondsToClockDuration(user_duration_format, time_to_read, false, true) or _("N/A") },
```

Comment:
Oh,  yes, when I saw that one, I was a bit confused.
It's shorter, but I find "Estimated reading finished" clearer (even if gramatically/sounding strange?)

Reply:
Mmm I can't think of a way to put read in there without being awkward